modules/Microsoft.EventGrid/topics/eventSubscriptions/deploy.bicep

```
@@ -61,9 +61,9 @@ resource eventGridTopic 'Microsoft.EventGrid/topics@2022-06-15' existing = {
   name: eventGridTopicName
 }
 
-resource eventSubscription 'Microsoft.EventGrid/eventSubscriptions@2022-06-15' = {
+resource eventSubscription 'Microsoft.EventGrid/topics/eventSubscriptions@2022-06-15' = {
   name: name
-  scope: eventGridTopic
+  parent: eventGridTopic
```

```suggestion
  parent: topic
```
To keep it crisp. However, that would also need updating in the existing resource above